monitor/avs/avs.go

```diff
@@ -0,0 +1,58 @@
+package avs
+
+import (
+	"context"
+
+	"github.com/omni-network/omni/contracts/bindings"
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/ethclient"
+	"github.com/omni-network/omni/lib/log"
+	"github.com/omni-network/omni/lib/netconf"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// Monitor starts monitoring the AVS contract.
+func Monitor(ctx context.Context, network netconf.Network, address common.Address) error {
+	log.Info(ctx, "Starting AVS monitor")
```

Comment:
You already log about this in the `Run` method.

Reply:
I log `Starting monitor service` there. avs monitoring would be one thing that service monitories. Though, it's the only right now